From 01e9eda09749b6270e5a34900e2e62f84c798736 Mon Sep 17 00:00:00 2001
From: Michal Galka <michal.galka@collabora.com>
Date: Wed, 28 Apr 2021 18:13:09 +0200
Subject: [PATCH 1/3] CHROMEOS Update configuration for chromeos.kernelci.org

There are paths and filenames that need to be adjusted for
chromeos.kernelci.org. This commit updates appropriate values.

Signed-off-by: Michal Galka <michal.galka@collabora.com>
---
 app/models/__init__.py  | 2 +-
 app/server.py           | 5 +++--
 app/taskqueue/celery.py | 3 ++-
 3 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/app/models/__init__.py b/app/models/__init__.py
index b0764510b71a..a445b4c7ca8b 100644
--- a/app/models/__init__.py
+++ b/app/models/__init__.py
@@ -27,7 +27,7 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
 # The default mongodb database name.
-DB_NAME = "kernel-ci"
+DB_NAME = "chromeos-kernel-ci"
 
 DEFAULT_SCHEMA_VERSION = "1.0"
 
diff --git a/app/server.py b/app/server.py
index 7faff9d842a5..ac75ed80747d 100755
--- a/app/server.py
+++ b/app/server.py
@@ -37,7 +37,8 @@ import utils.database.redisdb as redisdb
 import utils.db
 
 
-DEFAULT_CONFIG_FILE = "/etc/kernelci/kernelci-backend.cfg"
+DEFAULT_CONFIG_FILE = "/etc/kernelci/api.chromeos.kernelci.org/" \
+                      "kernelci-backend.cfg"
 
 topt.define(
     "master_key", default=str(uuid.uuid4()), type=str, help="The master key")
@@ -171,7 +172,7 @@ if __name__ == "__main__":
 
         server = tornado.httpserver.HTTPServer(application, **HTTP_SETTINGS)
         unix_socket = tornado.netutil.bind_unix_socket(
-            "/tmp/kernelci-backend.socket")
+            "/tmp/chromeos-kernelci-backend.socket")
         server.add_socket(unix_socket)
     else:
         KernelCiBackend().listen(topt.options.port, **HTTP_SETTINGS)
diff --git a/app/taskqueue/celery.py b/app/taskqueue/celery.py
index f2c694e0127d..5a4078e9a9e2 100644
--- a/app/taskqueue/celery.py
+++ b/app/taskqueue/celery.py
@@ -35,7 +35,8 @@ import taskqueue.celeryconfig as celeryconfig
 import taskqueue.serializer as serializer
 
 
-CELERY_CONFIG_FILE = "/etc/kernelci/kernelci-celery.cfg"
+CELERY_CONFIG_FILE = "/etc/kernelci/api.chromeos.kernelci.org/" \
+                     "kernelci-celery.cfg"
 TASKS_LIST = [
     "taskqueue.tasks.bisect",
     "taskqueue.tasks.build",
-- 
2.20.1

